hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/FileIoProvider.java

```diff
@@ -513,6 +513,7 @@ public boolean fullyDelete(@Nullable FsVolumeSpi volume, File dir) {
     try {
       faultInjectorEventHook.beforeMetadataOp(volume, DELETE);
       boolean deleted = FileUtil.fullyDelete(dir);
+      LOG.trace("Deletion of dir {} {}", dir, deleted ? "succeeded" : "failed");
```

Comment:
Do we need to add the trace logs as it is unrelated to the fix?

Reply:
While debugging this issue, I felt that a log line here would make it easier for operator to confirm the output of Delete operation, and trace log level is sufficient on the other hand, hence I thought of keeping this log line.